spec/unit/shoulda/matchers/doublespeak/double_spec.rb

```diff
@@ -92,50 +139,96 @@ module Shoulda::Matchers::Doublespeak
     end
 
     describe '#record_call' do
-      it 'stores the arguments and block given to the method in calls' do
-        double = described_class.new(:klass, :a_method, :implementation)
-        calls = [
-          [:any, :args], :block,
-          [:more, :args]
-        ]
-        double.record_call(calls[0][0], calls[0][1])
-        double.record_call(calls[1][0], nil)
-
-        expect(double.calls[0].args).to eq calls[0][0]
-        expect(double.calls[0].block).to eq calls[0][1]
-        expect(double.calls[1].args).to eq calls[1][0]
+      it 'adds the given call to the list of calls' do
+        double = described_class.new(
+          :a_world,
+          :a_klass,
+          :a_method,
+          :an_implementation
+        )
+        double.record_call(:some_call)
+        expect(double.calls.last).to eq :some_call
       end
     end
 
     describe '#call_original_method' do
-      it 'binds the stored method object to the class and calls it with the given args and block' do
-        klass = create_class
-        instance = klass.new
-        actual_args = actual_block = method_called = nil
+      it 'binds the stored method object to the given object and calls it with the given args and block' do
         expected_args = [:one, :two, :three]
         expected_block = -> { }
-        double = described_class.new(klass, :a_method, :implementation)
-
-        klass.__send__(:define_method, :a_method) do |*args, &block|
+        actual_args = actual_block = method_called = nil
+        method_name = :a_method
+        klass = create_class
+        klass.__send__(:define_method, method_name) do |*args, &block|
```

Comment:
Unused block argument - `args`. You can omit all the arguments if you don't care about them.<br>Unused block argument - `block`. You can omit all the arguments if you don't care about them.